features/step_definitions/ui/view-category-product.steps.js

```diff
@@ -0,0 +1,40 @@
+const { When, Then } = require("@cucumber/cucumber");
+const { expect } = require("@playwright/test");
+
+
+Then('I verify that categories are visible on left side bar', async function () {
+  const categoriesSidebar = await this.page.locator('.left-sidebar');
+  await expect(categoriesSidebar).toBeVisible();
+});
+
+When("I click on Women category", async function () {
```

Comment:
create generic step, we can use with each category.

"I click on {string} category"
- I click on Women category
- I click on Men category